test/Microsoft.AspNetCore.OData.Versioning.ApiExplorer.Tests/AspNet.OData/DefaultModelTypeBuilderTest.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace Microsoft.AspNet.OData
+﻿using System.Runtime.Serialization;
```

Comment:
Looks like VS wasn't _helpful_ and added the `using` statements in the wrong place. >_<